README.md

```diff
@@ -115,7 +115,7 @@ The `ada_language_server` doesn't require/understand any command line options.
 | `documentLink/resolve`                |                    |
 | `textDocument/documentColor`          |                    |
 | `textDocument/colorPresentation`      |                    |
-| `textDocument/formatting`             |                    |
+| `textDocument/formatting`             | :white_check_mark: |
 | `textDocument/rangeFormatting`        |                    |
```

Comment:
rangeFormatting has also ben implemented actually

Reply:
No, it's disabled now. We reverted `rangeFormatting` because it caused a few regressions in GS testsuite.